tests/TestCase/PluginTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2022 Atlas Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\DevTools\Test\TestCase;
+
+use BEdita\DevTools\Middleware\HtmlMiddleware;
+use BEdita\DevTools\Plugin;
+use Cake\Core\Configure;
+use Cake\Error\Middleware\ErrorHandlerMiddleware;
+use Cake\Http\BaseApplication;
+use Cake\Http\MiddlewareQueue;
+use Cake\Http\ServerRequest;
+use Cake\Routing\Middleware\AssetMiddleware;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test {@see \BEdita\DevTools\Plugin}.
+ *
+ * @coversDefaultClass \BEdita\DevTools\Plugin
+ */
+class PluginTest extends TestCase
+{
+    /**
+     * Test subject.
+     *
+     * @var \BEdita\DevTools\Plugin
+     */
+    protected $plugin;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->plugin = new Plugin();
+    }
+
+    /**
+     * Test {@see Plugin::bootstrap()} method.
+     *
+     * @return void
+     *
```

Comment:
[minor] remove empty lines between annotations (part of Cake 4 coding style rules)